### ISSUE TITLE
Current master fails to build

### ISSUE THREAD

**Issue by ycollet** [monocasual/giada#412]
I build the current master on Fedora 32 with gcc 10.2.1.
The current master fails to build with the following error message:
```
[  5%] Building CXX object CMakeFiles/giada.dir/src/glue/main.cpp.o
Dans le fichier inclus depuis /home/collette/repositories/giada/src/gui/elems/mainWindow/mainTimer.h:35,
                 depuis /home/collette/repositories/giada/src/glue/main.cpp:33:
/home/collette/repositories/giada/src/gui/elems/basics/choice.h:50:26: erreur: « string » dans l'espace de noms « std » ne nomme pas un type
   50 |  void addItem(const std::string& label, ID id);
      |                          ^~~~~~
/home/collette/repositories/giada/src/gui/elems/basics/choice.h:35:1: note: « std::string » est défini dans l'en-tête « <string> » ; avez-vous oublié « #include <string> » ?
   34 | #include <FL/Fl_Choice.H>

```

**Comment by tomek-szczesny** [comment]
It looks like `choice.h` lacks:
`#include <string>`
You could try adding it manually and see if problem persists.

Oddly enough, I have no such problems (and neither Travis CI). This might have something to do with deprecated building system - currently Giada is built using CMake instead of old makefile. If you followed the instructions on a website, those are outdated now. 
You may want to try building using CMake instead, if you didn't so. Let us know if you need help with that.

Nevertheless, the missing include probably should be added to the file.

**Comment by ycollet** [comment]
I added it and that was tthe only header missing

**Comment by ycollet** [comment]
I used cmake to build giada. I think this is a "problem" which comes with last gcc release. They are much sensitive to missing headers.

**Comment by tomek-szczesny** [comment]
Good - at least now we know there is one missing. ;)

Thanks for reporting!

**Comment by monocasual** [comment]
This should be fixed with PR #413 by @tomek-szczesny .

**Comment by ycollet** [comment]
Compiles fine now. I close the bug report.
